### DISCUSSION TITLE
Possible to monitor USB interface without DAW using black hole?

### DISCUSSION THREAD

**Discussion by mk347** [ExistentialAudio/BlackHole#659] [General]
I have a sequencer connected to USB interface. I want to hear everything from this sequencer through my pc speakers. I'm able to accomplish this using GarageBand and monitoring the recording in the DAW. I want to know if I can do this without using GarageBand or any DAW using Black Hole? I just want to hear my PC sounds and the sequencer sounds simultaneously without opening a program. Thanks!

**Comment by devinroth** [comment]
You need some sort of program if you want to do it on the PC side. But really you should just enable input monitoring on your interface.

**Reply by mk347** [reply]
I ended up doing just that. Connected my monitors to the interface output via a special Y cable, and now I can hear everything. Thanks for the timely response!
